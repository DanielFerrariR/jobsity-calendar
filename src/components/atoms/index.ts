export { default as Box } from './box'
export { default as Button } from './button'
export { default as Paper } from './paper'
export { default as Typography } from './typography'
export { default as TextField } from './text_field'
export { default as Container } from './container'
export { default as AppBar } from './appbar'
export { default as IconButton } from './icon_button'
export { default as Toolbar } from './toolbar'
export { default as Image } from './image'
export { default as Grid } from './grid'
export { default as Link } from './link'
export { default as Checkbox } from './checkbox'
export { default as Accordion } from './accordion'
export { default as AccordionSummary } from './accordion_summary'
export { default as AccordionDetails } from './accordion_details'
export { default as Tooltip } from './tooltip'
export { default as Modal } from './modal'
export { default as MenuItem } from './menu_item'
export { default as ListItemIcon } from './list_item_icon'
export { default as ListItemText } from './list_item_text'
export { default as ListItemSecondaryAction } from './list_item_secondary_action'
export { default as MenuList } from './menu_list'
export { default as Popover } from './popover'
export { default as Fade } from './fade'
export { default as Backdrop } from './backdrop'
export { default as CircularProgress } from './circular_progress'
export { default as LinearProgress } from './linear_progress'
export { default as Skeleton } from './skeleton'
export { default as List } from './list'
export { default as ListItem } from './list_item'
export { default as ClickAwayListener } from './click_away_listener'
export { default as Badge } from './badge'
export { default as Portal } from './portal'
export { default as Snackbar } from './snackbar'
export { default as Step } from './step'
export { default as StepContent } from './stepContent'
export { default as StepLabel } from './stepLabel'
export { default as Stepper } from './stepper'
export { default as TableBody } from './table_body'
export { default as TableContainer } from './table_container'
export { default as TableHead } from './table_head'
export { default as TableRow } from './table_row'
export { default as Table } from './table'
export { default as TableCell } from './table_cell'
export { default as DateTimePicker } from './date_time_picker'
