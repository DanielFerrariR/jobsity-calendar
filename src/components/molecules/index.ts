export { CustomTableHead, CustomTableBody } from './custom_table'
export { default as Modal } from './modal'
export { default as ReminderCard } from './reminder_card'
