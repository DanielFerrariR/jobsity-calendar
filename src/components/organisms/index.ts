export { default as Calendar } from './calendar'
export { default as ListRemindersModal } from './list_reminders_modal'
export { default as CreateReminderModal } from './create_reminder_modal'
export { default as EditReminderModal } from './edit_reminder_modal'
