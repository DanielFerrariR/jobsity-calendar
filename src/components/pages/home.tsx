import React from 'react'
import { Container } from 'src/components/atoms'
import { Calendar } from 'src/components/organisms'

const Home: React.FC = () => {
  return (
    <Container mt={4}>
      <Calendar />
    </Container>
  )
}

export default Home
