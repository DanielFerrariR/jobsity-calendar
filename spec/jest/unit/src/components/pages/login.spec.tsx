import React from 'react'
import { Login } from 'src/components/pages'
import { render } from 'spec/jest/config/test_utils'
import { axe } from 'jest-axe'

describe('testing Login component', () => {
  beforeEach(() => {
    jest.useFakeTimers()
  })

  afterEach(() => {
    jest.useRealTimers()
  })

  test('should snapshot works with english language', () => {
    const { container } = render(<Login />)

    expect(container).toMatchSnapshot()
  })

  test('should snapshot works with portuguese language', () => {
    const { container } = render(<Login />, {
      language: 'pt'
    })

    expect(container).toMatchSnapshot()
  })

  test('should snapshot works with spanish language', () => {
    const { container } = render(<Login />, {
      language: 'es'
    })

    expect(container).toMatchSnapshot()
  })

  test('should the page be accessible', async () => {
    const { container } = render(<Login />)

    jest.useRealTimers()

    const results = await axe(container)

    expect(results).toHaveNoViolations()
  })
})
