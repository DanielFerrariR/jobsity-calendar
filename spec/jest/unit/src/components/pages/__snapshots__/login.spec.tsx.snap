// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Login component should snapshot works with english language 1`] = `
<div>
  <div
    class="MuiContainer-root Component-root Component-root MuiContainer-maxWidthXs"
  >
    <img
      alt="Welcome logo"
      class="img-root img-root"
      src=""
    />
    <div
      class="MuiPaper-root Component-root Component-root MuiPaper-elevation1 MuiPaper-rounded"
    >
      <div
        class="MuiBox-root MuiBox-root Component-root Component-root"
      >
        <a
          aria-disabled="false"
          aria-label="Back"
          class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
          href="/front"
          role="button"
          tabindex="0"
          title="Back"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.41 16.59L10.83 12l4.58-4.59L14 6l-6 6 6 6 1.41-1.41z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
      </div>
      <div
        class="MuiBox-root MuiBox-root Component-root Component-root"
      >
        <h6
          class="MuiTypography-root Component-root Component-root MuiTypography-h6 MuiTypography-alignCenter"
        >
          Sign in
        </h6>
        <p
          class="MuiTypography-root Component-root Component-root MuiTypography-body1 MuiTypography-alignCenter"
        >
          Use e-mail and password used
        </p>
        <p
          class="MuiTypography-root Component-root Component-root MuiTypography-body1 MuiTypography-alignCenter"
        >
          when you created your account
        </p>
      </div>
      <form
        class="MuiBox-root MuiBox-root Component-root Component-root"
      >
        <div
          class="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root WithStyles(ForwardRef(TextField))-root"
        >
          <label
            class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-required MuiInputLabel-required"
            data-shrink="true"
            data-testid="login-label"
            for="textfield-login"
            id="textfield-login-label"
          >
            Username/e-mail
            <span
              aria-hidden="true"
              class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          >
            <input
              aria-invalid="false"
              autocomplete="off"
              class="MuiInputBase-input MuiOutlinedInput-input"
              data-testid="login"
              id="textfield-login"
              maxlength="100"
              placeholder="Enter your username or email"
              required=""
              type="text"
              value=""
            />
            <fieldset
              aria-hidden="true"
              class="PrivateNotchedOutline-root MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="PrivateNotchedOutline-legendLabelled PrivateNotchedOutline-legendNotched"
              >
                <span>
                  Username/e-mail
                   *
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <div
          class="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root WithStyles(ForwardRef(TextField))-root"
        >
          <label
            class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-required MuiInputLabel-required"
            data-shrink="true"
            data-testid="password-label"
            for="textfield-password"
            id="textfield-password-label"
          >
            Password
            <span
              aria-hidden="true"
              class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          >
            <input
              aria-invalid="false"
              autocomplete="off"
              class="MuiInputBase-input MuiOutlinedInput-input"
              data-testid="password"
              id="textfield-password"
              maxlength="100"
              placeholder="Enter your password"
              required=""
              type="password"
              value=""
            />
            <fieldset
              aria-hidden="true"
              class="PrivateNotchedOutline-root MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="PrivateNotchedOutline-legendLabelled PrivateNotchedOutline-legendNotched"
              >
                <span>
                  Password
                   *
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root Component-root Component-root"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained Component-root Component-root MuiButton-containedSecondary MuiButton-containedSizeLarge MuiButton-sizeLarge"
            data-testid="submit-button"
            tabindex="0"
            type="submit"
          >
            <span
              class="MuiButton-label"
            >
              Sign in
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
        <button
          class="MuiTypography-root MuiLink-root MuiLink-underlineHover Component-root Component-root MuiLink-button MuiTypography-subtitle2 MuiTypography-colorPrimary"
          type="button"
        >
          Forgot your password?
        </button>
        <p
          class="MuiTypography-root Component-root Component-root MuiTypography-body2 MuiTypography-alignCenter"
        >
          Don't have an account?
           
          <a
            class="MuiTypography-root MuiLink-root MuiLink-underlineHover Component-root Component-root MuiTypography-subtitle2 MuiTypography-colorPrimary"
            href="/front/register"
          >
            Create account
          </a>
        </p>
      </form>
    </div>
  </div>
  <div
    class="MuiBox-root MuiBox-root Component-root Component-root"
  >
    <p
      class="MuiTypography-root Component-root Component-root MuiTypography-body2"
    >
      Created
       
      <a
        class="MuiTypography-root MuiLink-root MuiLink-underlineHover Component-root Component-root MuiTypography-subtitle2 MuiTypography-colorPrimary"
        href="https://autoseg.com"
      >
        @AutoSeg
      </a>
    </p>
    <p
      class="MuiTypography-root Component-root Component-root MuiTypography-body2"
    >
      Version
       1.0.0
    </p>
    <div
      class="MuiBox-root MuiBox-root Component-root Component-root"
    >
      <button
        aria-label="Portuguese"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        data-testid="change-language-portuguese"
        tabindex="0"
        title="Portuguese"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <img
            alt="Brasil flag"
            class="img-root img-root"
            src=""
          />
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <button
        aria-label="English"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        data-testid="change-language-english"
        tabindex="0"
        title="English"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <img
            alt="United Kingdom flag"
            class="img-root img-root"
            src=""
          />
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <button
        aria-label="Spanish"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        data-testid="change-language-spanish"
        tabindex="0"
        title="Spanish"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <img
            alt="Spain flag"
            class="img-root img-root"
            src=""
          />
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
</div>
`;

exports[`testing Login component should snapshot works with portuguese language 1`] = `
<div>
  <div
    class="MuiContainer-root Component-root Component-root MuiContainer-maxWidthXs"
  >
    <img
      alt="Logotipo de bem-vindo"
      class="img-root img-root"
      src=""
    />
    <div
      class="MuiPaper-root Component-root Component-root MuiPaper-elevation1 MuiPaper-rounded"
    >
      <div
        class="MuiBox-root MuiBox-root Component-root Component-root"
      >
        <a
          aria-disabled="false"
          aria-label="Voltar"
          class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
          href="/front"
          role="button"
          tabindex="0"
          title="Voltar"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.41 16.59L10.83 12l4.58-4.59L14 6l-6 6 6 6 1.41-1.41z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
      </div>
      <div
        class="MuiBox-root MuiBox-root Component-root Component-root"
      >
        <h6
          class="MuiTypography-root Component-root Component-root MuiTypography-h6 MuiTypography-alignCenter"
        >
          Logar
        </h6>
        <p
          class="MuiTypography-root Component-root Component-root MuiTypography-body1 MuiTypography-alignCenter"
        >
          Digite o e-mail e senha usados
        </p>
        <p
          class="MuiTypography-root Component-root Component-root MuiTypography-body1 MuiTypography-alignCenter"
        >
          quando você criou sua conta
        </p>
      </div>
      <form
        class="MuiBox-root MuiBox-root Component-root Component-root"
      >
        <div
          class="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root WithStyles(ForwardRef(TextField))-root"
        >
          <label
            class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-required MuiInputLabel-required"
            data-shrink="true"
            data-testid="login-label"
            for="textfield-login"
            id="textfield-login-label"
          >
            Nome de usuário/e-mail
            <span
              aria-hidden="true"
              class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          >
            <input
              aria-invalid="false"
              autocomplete="off"
              class="MuiInputBase-input MuiOutlinedInput-input"
              data-testid="login"
              id="textfield-login"
              maxlength="100"
              placeholder="Digite o nome ou e-mail"
              required=""
              type="text"
              value=""
            />
            <fieldset
              aria-hidden="true"
              class="PrivateNotchedOutline-root MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="PrivateNotchedOutline-legendLabelled PrivateNotchedOutline-legendNotched"
              >
                <span>
                  Nome de usuário/e-mail
                   *
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <div
          class="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root WithStyles(ForwardRef(TextField))-root"
        >
          <label
            class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-required MuiInputLabel-required"
            data-shrink="true"
            data-testid="password-label"
            for="textfield-password"
            id="textfield-password-label"
          >
            Senha
            <span
              aria-hidden="true"
              class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          >
            <input
              aria-invalid="false"
              autocomplete="off"
              class="MuiInputBase-input MuiOutlinedInput-input"
              data-testid="password"
              id="textfield-password"
              maxlength="100"
              placeholder="Digite sua senha"
              required=""
              type="password"
              value=""
            />
            <fieldset
              aria-hidden="true"
              class="PrivateNotchedOutline-root MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="PrivateNotchedOutline-legendLabelled PrivateNotchedOutline-legendNotched"
              >
                <span>
                  Senha
                   *
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root Component-root Component-root"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained Component-root Component-root MuiButton-containedSecondary MuiButton-containedSizeLarge MuiButton-sizeLarge"
            data-testid="submit-button"
            tabindex="0"
            type="submit"
          >
            <span
              class="MuiButton-label"
            >
              Logar
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
        <button
          class="MuiTypography-root MuiLink-root MuiLink-underlineHover Component-root Component-root MuiLink-button MuiTypography-subtitle2 MuiTypography-colorPrimary"
          type="button"
        >
          Esqueceu sua senha?
        </button>
        <p
          class="MuiTypography-root Component-root Component-root MuiTypography-body2 MuiTypography-alignCenter"
        >
          Não tem uma conta?
           
          <a
            class="MuiTypography-root MuiLink-root MuiLink-underlineHover Component-root Component-root MuiTypography-subtitle2 MuiTypography-colorPrimary"
            href="/front/register"
          >
            Criar conta
          </a>
        </p>
      </form>
    </div>
  </div>
  <div
    class="MuiBox-root MuiBox-root Component-root Component-root"
  >
    <p
      class="MuiTypography-root Component-root Component-root MuiTypography-body2"
    >
      Criado
       
      <a
        class="MuiTypography-root MuiLink-root MuiLink-underlineHover Component-root Component-root MuiTypography-subtitle2 MuiTypography-colorPrimary"
        href="https://autoseg.com"
      >
        @AutoSeg
      </a>
    </p>
    <p
      class="MuiTypography-root Component-root Component-root MuiTypography-body2"
    >
      Versão
       1.0.0
    </p>
    <div
      class="MuiBox-root MuiBox-root Component-root Component-root"
    >
      <button
        aria-label="Português"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        data-testid="change-language-portuguese"
        tabindex="0"
        title="Português"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <img
            alt="Bandeira do Brasil"
            class="img-root img-root"
            src=""
          />
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <button
        aria-label="Inglês"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        data-testid="change-language-english"
        tabindex="0"
        title="Inglês"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <img
            alt="Bandeira do Reino Unido"
            class="img-root img-root"
            src=""
          />
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <button
        aria-label="Espanhol"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        data-testid="change-language-spanish"
        tabindex="0"
        title="Espanhol"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <img
            alt="Bandeira da Espanha"
            class="img-root img-root"
            src=""
          />
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
</div>
`;

exports[`testing Login component should snapshot works with spanish language 1`] = `
<div>
  <div
    class="MuiContainer-root Component-root Component-root MuiContainer-maxWidthXs"
  >
    <img
      alt="Logotipo de Bienvenido"
      class="img-root img-root"
      src=""
    />
    <div
      class="MuiPaper-root Component-root Component-root MuiPaper-elevation1 MuiPaper-rounded"
    >
      <div
        class="MuiBox-root MuiBox-root Component-root Component-root"
      >
        <a
          aria-disabled="false"
          aria-label="Volver"
          class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
          href="/front"
          role="button"
          tabindex="0"
          title="Volver"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.41 16.59L10.83 12l4.58-4.59L14 6l-6 6 6 6 1.41-1.41z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </a>
      </div>
      <div
        class="MuiBox-root MuiBox-root Component-root Component-root"
      >
        <h6
          class="MuiTypography-root Component-root Component-root MuiTypography-h6 MuiTypography-alignCenter"
        >
          Iniciar sesión
        </h6>
        <p
          class="MuiTypography-root Component-root Component-root MuiTypography-body1 MuiTypography-alignCenter"
        >
          Utilice el e-mail y la contraseña
        </p>
        <p
          class="MuiTypography-root Component-root Component-root MuiTypography-body1 MuiTypography-alignCenter"
        >
          utilizada cuando creaste tu cuenta
        </p>
      </div>
      <form
        class="MuiBox-root MuiBox-root Component-root Component-root"
      >
        <div
          class="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root WithStyles(ForwardRef(TextField))-root"
        >
          <label
            class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-required MuiInputLabel-required"
            data-shrink="true"
            data-testid="login-label"
            for="textfield-login"
            id="textfield-login-label"
          >
            Nombre de usuario/e-mail
            <span
              aria-hidden="true"
              class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          >
            <input
              aria-invalid="false"
              autocomplete="off"
              class="MuiInputBase-input MuiOutlinedInput-input"
              data-testid="login"
              id="textfield-login"
              maxlength="100"
              placeholder="Ingrese el nombre o e-mail"
              required=""
              type="text"
              value=""
            />
            <fieldset
              aria-hidden="true"
              class="PrivateNotchedOutline-root MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="PrivateNotchedOutline-legendLabelled PrivateNotchedOutline-legendNotched"
              >
                <span>
                  Nombre de usuario/e-mail
                   *
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <div
          class="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root WithStyles(ForwardRef(TextField))-root"
        >
          <label
            class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-required MuiInputLabel-required"
            data-shrink="true"
            data-testid="password-label"
            for="textfield-password"
            id="textfield-password-label"
          >
            Contraseña
            <span
              aria-hidden="true"
              class="MuiFormLabel-asterisk MuiInputLabel-asterisk"
            >
               
              *
            </span>
          </label>
          <div
            class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
          >
            <input
              aria-invalid="false"
              autocomplete="off"
              class="MuiInputBase-input MuiOutlinedInput-input"
              data-testid="password"
              id="textfield-password"
              maxlength="100"
              placeholder="Ingresa tu contraseña"
              required=""
              type="password"
              value=""
            />
            <fieldset
              aria-hidden="true"
              class="PrivateNotchedOutline-root MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="PrivateNotchedOutline-legendLabelled PrivateNotchedOutline-legendNotched"
              >
                <span>
                  Contraseña
                   *
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
        <div
          class="MuiBox-root MuiBox-root Component-root Component-root"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained Component-root Component-root MuiButton-containedSecondary MuiButton-containedSizeLarge MuiButton-sizeLarge"
            data-testid="submit-button"
            tabindex="0"
            type="submit"
          >
            <span
              class="MuiButton-label"
            >
              Iniciar sesión
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
        <button
          class="MuiTypography-root MuiLink-root MuiLink-underlineHover Component-root Component-root MuiLink-button MuiTypography-subtitle2 MuiTypography-colorPrimary"
          type="button"
        >
          ¿Olvidaste tu contraseña?
        </button>
        <p
          class="MuiTypography-root Component-root Component-root MuiTypography-body2 MuiTypography-alignCenter"
        >
          ¿No tienes una cuenta?
           
          <a
            class="MuiTypography-root MuiLink-root MuiLink-underlineHover Component-root Component-root MuiTypography-subtitle2 MuiTypography-colorPrimary"
            href="/front/register"
          >
            Crear cuenta
          </a>
        </p>
      </form>
    </div>
  </div>
  <div
    class="MuiBox-root MuiBox-root Component-root Component-root"
  >
    <p
      class="MuiTypography-root Component-root Component-root MuiTypography-body2"
    >
      Creado
       
      <a
        class="MuiTypography-root MuiLink-root MuiLink-underlineHover Component-root Component-root MuiTypography-subtitle2 MuiTypography-colorPrimary"
        href="https://autoseg.com"
      >
        @AutoSeg
      </a>
    </p>
    <p
      class="MuiTypography-root Component-root Component-root MuiTypography-body2"
    >
      Versión
       1.0.0
    </p>
    <div
      class="MuiBox-root MuiBox-root Component-root Component-root"
    >
      <button
        aria-label="Portugués"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        data-testid="change-language-portuguese"
        tabindex="0"
        title="Portugués"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <img
            alt="Bandera de Brasil"
            class="img-root img-root"
            src=""
          />
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <button
        aria-label="Inglés"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        data-testid="change-language-english"
        tabindex="0"
        title="Inglés"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <img
            alt="Bandera del Reino Unido"
            class="img-root img-root"
            src=""
          />
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
      <button
        aria-label="Español"
        class="MuiButtonBase-root MuiIconButton-root Component-root Component-root Styled(Component)-root Styled(Component)-root WithStyles(ForwardRef(Tooltip))-root WithStyles(ForwardRef(Tooltip))-root"
        data-testid="change-language-spanish"
        tabindex="0"
        title="Español"
        type="button"
      >
        <span
          class="MuiIconButton-label"
        >
          <img
            alt="Bandera de España"
            class="img-root img-root"
            src=""
          />
        </span>
        <span
          class="MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
</div>
`;
